control-plane/pkg/client/clientset/versioned/fake/register.go

```diff
@@ -37,14 +37,14 @@ var localSchemeBuilder = runtime.SchemeBuilder{
 // AddToScheme adds all types of this clientset into the given scheme. This allows composition
 // of clientsets, like in:
 //
-//	import (
```

Comment:
what's up with this? :smile:

Reply:
You have to upgrade to go 1.19.3 (gofmt included) , have you?

/hold

Reply:
All I really did was issuing:

```
make RELEASE=v1.7 generate-release
```

Nothing else

Reply:
I have 1.18.x on my box, and I see 1.18 referenced here: https://github.com/matzew/eventing-kafka-broker/blob/run_make_RELEASE_1.7/go.mod#L3 

Not sure more is needed - or why?

Reply:
Ok, I did not see this from upstream:
https://github.com/knative-sandbox/eventing-kafka-broker/pull/2770

Hence, I had not clue - just noticed this by accident...

Reply:
1.18 is correct in go.mod and it will work, however, gofmt is gonna produce different files based on its version, that's why I'd move to Go 1.19 since it required upstream as well for development

Reply:
This is the why (from https://tip.golang.org/doc/go1.19):
```
Doc Comments
Go 1.19 adds support for links, lists, and clearer headings in doc comments. As part of this change, [gofmt](https://tip.golang.org/cmd/gofmt) now reformats doc comments to make their rendered meaning clearer. See “[Go Doc Comments](https://tip.golang.org/doc/comment)” for syntax details and descriptions of common mistakes now highlighted by gofmt. As another part of this change, the new package [go/doc/comment](https://tip.golang.org/pkg/go/doc/comment/) provides parsing and reformatting of doc comments as well as support for rendering them to HTML, Markdown, and text.
```